dynamic_rest/fields/generic.py

```diff
@@ -84,7 +84,7 @@ def to_representation(self, instance):
             # We want the pk to be represented as an object with type,
             # rather than just the ID.
             pk_value = self.get_pk_object(
-                serializer_class().get_plural_name(),
+                serializer_class().get_name(),
```

Comment:
`get_plural_name` is also a class method, it seems.  I should change that one as well, correct?